### PR TITLE
Fixes for 2.3 branch

### PR DESCRIPTION
**Pull request by ifdattic**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.3 |
| Fixed tickets |  |

### CODE REVIEW

File: components/http_foundation/sessions.rst [hunk 1]
```diff
@@ -190,8 +190,12 @@ data is an array, for example a set of tokens. In this case, managing the array
 becomes a burden because you have to retrieve the array then process it and
 store it again::
 
-    $tokens = array('tokens' => array('a' => 'a6c1e0b6',
-                                      'b' => 'f4a7b1f3'));
+    $tokens = array(
+        'tokens' => array(
+            'a' => 'a6c1e0b6',
+            'b' => 'f4a7b1f3',
+        )
```
Comment (xabbuh) [inline components/http_foundation/sessions.rst]: missing comma here

Comment (xabbuh) [inline components/http_foundation/sessions.rst]: We could also use this:

```php
$tokens = array('tokens' => array(
    'a' => 'a6c1e0b6',
    'b' => 'f4a7b1f3',
));
```

What do you think?

Comment (WouterJ) [inline components/http_foundation/sessions.rst]: -1 for that proposal
+1 for the comma

I think the key is a bit lost in the proposal and it isn't very clear that it's a tree structure and a and b are childs of tokens.

Comment (ifdattic) [inline components/http_foundation/sessions.rst]: Yes sorry, missed that comma, will fix soon.

At first I also did as @xabbuh suggested, but decided against it as it doesn't make it as clear.

Comment (xabbuh) [inline components/http_foundation/sessions.rst]: Indeed, this is more readable than what I suggested.

**Pull request merged**